WeakAuras/Prototypes.lua

```diff
@@ -2282,6 +2296,7 @@ Private.event_prototypes = {
   },
   ["Health"] = {
     type = "unit",
+    includePets = "true",
```

Comment:
A few changes here, and below so that triggers that don't support pets are handled the same as before, even if there might be strange settings.